apps/frontend/src/app/Types/character.d.ts

```diff
@@ -21,7 +21,7 @@ import type { IConditionalValues } from './sheet'
 export interface CustomTarget {
   weight: number
   path: string[]
-  hitMode: HitModeKey
```

Comment:
I'd prefer to have a `MultiOptHitModeKey` with 'global' added so it can simplify this, the validation code for multi-opts, and the UI code as well

```typescript
const allMultiOptHitModeKeys = [...allHitModeKeys, 'global']
```

Reply:
I've implemented this now